src/app-layout/visual-refresh/main.scss

```diff
@@ -129,7 +129,9 @@
 }
 
 // Prevent content that is visually hidden behind drawers and content area in mobile view from receiving focus
-/* stylelint-disable-next-line selector-max-universal, selector-combinator-disallowed-list */
+/* stylelint-disable selector-max-universal, selector-combinator-disallowed-list */
+.unfocusable,
```

Comment:
Without this change, VoiceOver would still announce the regions (but not the content within them).